#ifndef CUSTOM_MESSAGE_H_
#define CUSTOM_MESSAGE_H_

#include "ibeo_driver/LidarMessage.h"
#include "ibeo_driver/AbsoluteVelocity.h"
#include "ibeo_driver/BoundingBox.h"
#include "ibeo_driver/MountingPosition.h"
#include "ibeo_driver/Object.h"
#include "ibeo_driver/ObjectBox.h"
#include "ibeo_driver/ObjectData.h"
#include "ibeo_driver/Point2D.h"
#include "ibeo_driver/ReferencePoint.h"
#include "ibeo_driver/ScanData.h"
#include "ibeo_driver/ScanPoint.h"
#include "ibeo_driver/Size2D.h"


#endif
